npm-react/react.js

```diff
@@ -1 +1,53 @@
-module.exports = require('./lib/React');
+'use strict';
+
+var React = require('./lib/React');
+
+var assign = require('./lib/Object.assign');
+var deprecated = require('./lib/deprecated');
+
```

Comment:
Can we initialize deprecations to null instead of relying on weird scope rules? Worst case some module system edge case would have a deprecations variable in scope that get reused.